### PR TITLE
Split CI for release 2020-06

### PR DESCRIPTION
**Pull request by ziyiwu9494**
Test alternate fix for #2053

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -15,10 +15,12 @@ on:
   push:
     branches:
       - master
-      - 'release-.*'
     tags:
       - 'v[1-9][0-9]*.[0-9]+.[0-9]+'
       - 'v[1-9][0-9]*.[0-9]+.[0-9]+rc[0-9]'
+# Make sure to ignore all current release tags
```
Comment (ryanjulian) [inline .github/workflows/ci.yml:21]: might as well ignore *all* release tags instead

**Review by ryanjulian — APPROVED** [review]
See my 1 comment. LGTM if this is working!

File: .github/workflows/ci.yml (hunk 1 above)
Comment (ziyiwu9494) [reply .github/workflows/ci.yml:21]: Do we never deploy any tags from master? I don't have a regex for all possible release tags.

Comment (ryanjulian) [reply .github/workflows/ci.yml:21]: in practice i think all release tags will match the ones you have above. we might deploy from master one day soon (for nightly builds), but we can implement that later. just get this one working first.

**Pull request merged**